g3w-admin/base/urls.py

```diff
@@ -143,6 +144,7 @@
 
 register_i18n_trees(G3W_SITETREE_I18N_ALIAS)
 
-
-
-
+if settings.DEBUG:
+    urlpatterns += [
+        re_path(r'^static/(?P<path>.*)$', views.serve),
```

Comment:
@elpaso my dev installation just works without it.

Reply:
My gunicorn doesn't :)